allennlp/data/fields/tensor_field.py

```diff
@@ -76,3 +76,11 @@ def __eq__(self, other) -> bool:
     def array(self):
         """This is a compatibility method that returns the underlying tensor as a numpy array."""
         return self.tensor.numpy()
+
+    @overrides
+    def human_readable_dict(self):
+        size = list(self.tensor.shape)
+        std = torch.std(self.tensor.float()).item()
+        ave = torch.mean(self.tensor.float()).item()
+
+        return {"tensor": {"size": size, "element_std": std, "element_ave": ave}}
```

Comment:
I recommend just keeping the "shape" terminology:

```suggestion
        shape = list(self.tensor.shape)
        std = torch.std(self.tensor.float()).item()
        ave = torch.mean(self.tensor.float()).item()
        return {"tensor": {"shape": shape, "element_std": std, "element_ave": ave}}
```